data/stylish-haskell.yaml

```diff
@@ -31,8 +31,14 @@ steps:
   #     # See `separate_lists` for the `imports` step.
   #     separate_lists: true
   #
-  #     # Whether to break the "where" if there are no exports.
-  #     break_only_where: false
+  #     # When to break the "where".
+  #     # Possible values:
+  #     # - exports: only break when there is an explicit export list.
+  #     # - inline: only break when the export list is too long. This is
+  #     #   determined by the `columns` setting. Not applicable when the export
+  #     #   list contains comments as newlines will be required.
```

Comment:
If desired, I can add a `space_surround` option to toggle between the following two styles:
```haskell
module Foo () where
module Foo (main) where
module Foo (main, bar) where
```
```haskell
module Foo ( ) where -- or () ?
module Foo ( main ) where
module Foo ( main, bar ) where
```
The option would only apply in case of `inline`.

Reply:
Instead of `inline` (or in addition to?), I could also have something like `single`, which uses single-line formatting only when there are 0 to 1 exports. Starting from 2, multi-line formatting will be used.